src/gax-internal/src/http.rs

```diff
@@ -356,4 +369,22 @@ mod test {
         assert_eq!(err.headers(), &Some(want));
         Ok(())
     }
+
+    #[tokio::test]
+    async fn client_empty_content() -> TestResult {
+        let http_resp = http::Response::builder()
+            .header("Content-Type", "application/json")
+            .status(204)
+            .body(r#""#)?;
+        let response: reqwest::Response = http_resp.into();
+        assert!(response.status().is_success());
+
+        let response = ReqwestClient::to_http_response::<wkt::Empty>(response).await;
+        assert!(response.is_ok());
+
+        let response = response.unwrap();
+        let body = response.into_body();
+        assert_eq!(body, wkt::Empty::default());
+        Ok(())
+    }
```

Comment:
Do we have a test for 200 with body == `{}`?  What about a 204 with a non-empty body?